index.html

```diff
@@ -0,0 +1,533 @@
+            <span class="small-text">$33.99</span>
+          </div>
+        </article>
+        <article class="latest-news">
+          <div class="col-1">
+            <img src="./images/sale_14523_primary_image.jpg" alt="" class="main-img">
+          </div>
+          <div class="col-2">
+            <h3 class="black-heading">FastestVPN: Lifetime Subscription (5 Devices)</h3>
+            <span class="small-text">$19.99</span>
+          </div>
+        </article>
+        <article class="latest-news">
+          <div class="col-1">
+            <img src="./images/better.jpg" alt="" class="main-img">
+          </div>
+          <div class="col-2">
+            <h3 class="black-heading">BetterMe Home Workout & Diet: Lifetime Subscription</h3>
+            <span class="small-text">$39.99</span>
+          </div>
+        </article>
+        <article class="latest-news">
+          <div class="col-1">
+            <img src="./images/certificat.jpg" alt="" class="main-img">
+          </div>
+          <div class="col-2">
+            <h3 class="black-heading">The Complete 2020 CompTIA Certification Training Bundle</h3>
+            <span class="small-text">$69</span>
+          </div>
+        </article>
+        <article class="latest-news">
+          <div class="col-1">
+            <img src="./images/laptop.jpg" alt="" class="main-img">
+          </div>
+          <div class="col-2">
+            <h3 class="black-heading">Password Boss Premium: Lifetime Subscription (Unlimited Devices)</h3>
+            <span class="small-text">$49.99</span>
+          </div>
+        </article>
+        <article class="latest-news">
+          <div class="col-1">
+            <img src="./images/mac.jpg" alt="" class="main-img">
+          </div>
+          <div class="col-2">
+            <h3 class="black-heading">DIY Plan: Lifetime Subscription (Unlimited Videos)</h3>
+            <span class="small-text">$29.99</span>
+          </div>
+        </article>
+      </div>
+    </div>
+  </main>
+  <footer>
+    <div class="footer-icons">
+      <ul>
+        <li class="link-icons">
+          <a href="#"><i class="icon-cicle fab fa-facebook-f"></i></a>
+          <a href="#"><i class="icon-cicle fab fa-twitter"></i></a>
+          <a href="#"><i class="icon-cicle fa fa-youtube-play"></i></a>
+          <a href="#"><i class="icon-cicle fa fa-instagram"></i></a>
+          <a href="#"><i class="icon-cicle fa fa-pinterest"></i></a>
+          <a href="#"><i class="icon-cicle fas fa-envelope"></i></a>
+          <a href="#"><i class="icon-cicle fa fa-wifi"></i></a>
+        </li>
+        <li class="reach-out">
+          <a href="">EVENTS</a>
+          <a href="">ABOUT</a>
+          <a href="">ADVERTISE</a>
+          <a href="">JOBS</a>
+          <a href="">CONTACT</a>
+      </ul>
+    </div>
+    <div class="footer-bottom">
+      <p>&copy; 2006-2020 The Next Web B.V. Made with in Amsterdam.
+        Powered by StackPath</p>
+    </div>
```

Comment:
- [ ] The footer contains TNW logo with a copyright paragraph ([link to page](https://thenextweb.com/)). But the logo is missing in your version. Please add it.

![image](https://user-images.githubusercontent.com/60184961/93029307-cb65f880-f622-11ea-804d-44391d01490d.png)